sdk/core/abort-controller/README.md

```diff
@@ -1,61 +1,67 @@
-# Azure Core Aborter library for JS
+# Azure Abort Controller library for JS
 
-`Aborter` is used by the Azure SDK for JavaScript to abort pending work based on timeouts or other signals. `Aborter` is compatible with the `AbortSignal` used by the `fetch` API built into modern browsers.
+`AborterController` is used by the Azure SDK for JavaScript to abort pending work based on timeouts or other signals.
```

Comment:
Does `AborterController` here refer to the package or the class?

If it is the class, then isnt `AborterController` used by the users and the SDK use the AbortSignalLike? What I am saying is that we may want to re-word this to something like

> Use `AborterController` to create a `AbortSignal` which can then be passed to Azure SDKs to abort pending work based ....

or

> `AborterController` is used to create a `AbortSignal` to be passed to Azure SDKs to abort pending work ....

Reply:
This should actually be about the package, not the class. I’ll reword it so it’s clear this is the `@azure/abort-controller` package that I’m referring to, not the actual class.

Reply:
I updated this section to focus on the package, and provided more details on the classes in the Key Concepts section.